src/Nox.Types.Extensions/Types/LanguageCode/LanguageCodeExtensions.cs

```diff
@@ -0,0 +1,20 @@
+using Nox.Reference;
+
+namespace Nox.Types.Extensions;
+
+/// <summary>
+/// Provides extension methods for the LanguageCode
+/// </summary>
+public static class LanguageCodeExtensions
+{
+    /// <summary>
+    /// Retrieves the reference language for a given language code.
+    /// </summary>
+    /// <param name="languageCode">The language code.</param>
+    /// <returns>The reference language that corresponds to the given language code.</returns>
+    public static Language GetReferenceLanguage(this LanguageCode languageCode)
+        => World.Languages.GetByIso_639_1(languageCode.Value)!;
+
```

Comment:
Can we remove these trailing empty lines, please?

Reply:
Desired changes have been applied